src/Megamix/Patches.cpp

```diff
@@ -0,0 +1,125 @@
+        /* E0 */ MuseumRow({ 0x023, 0x05b, 0x006, 0x00e, 0x05d }, (char *) 0x50201b, 0, 0),
+
+        /* 0  */ MuseumRow({ 0x059, 0x005, 0x007, 0x00d, 0x101 }, (char *) 0x50215a, 0, 0),
+        /* 1  */ MuseumRow({ 0x002, 0x003, 0x00a, 0x00b, 0x101 }, (char *) 0x50217e, 0, 1),
+        /* 2  */ MuseumRow({ 0x069, 0x101, 0x101, 0x101, 0x101 }, (char *) 0x502141, 0, 2),
+        /* 3  */ MuseumRow({ 0x000, 0x006, 0x008, 0x00c, 0x101 }, (char *) 0x5021a2, 0, 3),
+        /* 4  */ MuseumRow({ 0x017, 0x01d, 0x02d, 0x044, 0x101 }, (char *) 0x5021bb, 0, 4),
+        /* 5  */ MuseumRow({ 0x068, 0x101, 0x101, 0x101, 0x101 }, (char *) 0x502165, 0, 5),
+        /* 6  */ MuseumRow({ 0x001, 0x004, 0x009, 0x00e, 0x101 }, (char *) 0x5021c6, 0, 6),
+        /* 7  */ MuseumRow({ 0x019, 0x01f, 0x02e, 0x045, 0x101 }, (char *) 0x5021d1, 0, 7),
+        /* 8  */ MuseumRow({ 0x06b, 0x101, 0x101, 0x101, 0x101 }, (char *) 0x502189, 0, 8),
+        /* 9  */ MuseumRow({ 0x00f, 0x025, 0x034, 0x04a, 0x05e }, (char *) 0x502033, 0, 8),
+        /* 10 */ MuseumRow({ 0x05a, 0x027, 0x02c, 0x049, 0x060 }, (char *) 0x501ff3, 0, 0),
+        /* 11 */ MuseumRow({ 0x012, 0x022, 0x038, 0x046, 0x061 }, (char *) 0x501ffb, 0, 1),
+        /* 12 */ MuseumRow({ 0x010, 0x028, 0x032, 0x047, 0x062 }, (char *) 0x502003, 0, 2),
+        /* 13 */ MuseumRow({ 0x018, 0x024, 0x037, 0x043, 0x063 }, (char *) 0x50200b, 0, 3),
+        /* 14 */ MuseumRow({ 0x011, 0x026, 0x035, 0x04d, 0x064 }, (char *) 0x502013, 0, 4),
+        /* 15 */ MuseumRow({ 0x01b, 0x023, 0x036, 0x04b, 0x065 }, (char *) 0x50201b, 0, 5),
+        /* 16 */ MuseumRow({ 0x01c, 0x021, 0x03c, 0x048, 0x101 }, (char *) 0x50214f, 1, 0),
+        /* 17 */ MuseumRow({ 0x014, 0x02a, 0x03f, 0x042, 0x101 }, (char *) 0x502173, 1, 1),
+        /* 18 */ MuseumRow({ 0x01a, 0x01e, 0x02f, 0x04c, 0x101 }, (char *) 0x502197, 1, 2),
+        /* 19 */ MuseumRow({ 0x06a, 0x101, 0x101, 0x101, 0x101 }, (char *) 0x5021ad, 1, 3),
+        /* 20 */ MuseumRow({ 0x04e, 0x052, 0x056, 0x057, 0x066 }, (char *) 0x502023, 0, 6),
+        /* 21 */ MuseumRow({ 0x050, 0x051, 0x054, 0x058, 0x067 }, (char *) 0x50202b, 0, 7),
+        /* 22 */ MuseumRow({ 0x04f, 0x053, 0x055, 0x05d, 0x05f }, (char *) 0x50203c, 0, 9),
+        /* 23 */ MuseumRow({ 0x013, 0x015, 0x016, 0x101, 0x101 }, (char *) 0x50d291, 0, 0),
+        /* 24 */ MuseumRow({ 0x020, 0x029, 0x05b, 0x101, 0x101 }, (char *) 0x50d29b, 0, 0),
+        /* 25 */ MuseumRow({ 0x02b, 0x030, 0x031, 0x101, 0x101 }, (char *) 0x50d265, 0, 0),
+        /* 26 */ MuseumRow({ 0x033, 0x039, 0x03a, 0x101, 0x101 }, (char *) 0x50d270, 0, 0),
+        /* 27 */ MuseumRow({ 0x03b, 0x03d, 0x03e, 0x101, 0x101 }, (char *) 0x50d27b, 0, 0),
+        /* 28 */ MuseumRow({ 0x040, 0x041, 0x05c, 0x101, 0x101 }, (char *) 0x50d286, 0, 0),
+    };
+
+    const MuseumRowColor MuseumRowColors[MuseumRowCount] {
+        /* E2 */ MuseumRowColor({ 0x42, 0x42, 0x42, 0xFF }, { 0x00, 0x00, 0x00, 0x00 }),
```

Comment:
might be worth making a macro here like `rgba(0x424242FF)` just for a tiny bit more space